### ISSUE TITLE
AI and Machine Learning - Replace "overlays" as a section

### ISSUE THREAD

**Issue by aardrian** [w3c/ai-accessibility#16]
[Section 5](https://raw.githack.com/w3c/ai-accessibility/FPWD-ai-accessbility/index.html#accessibility-overlays) is a top-level section devoted to a product category called "accessibility overlays."

I don't think it belongs in this document.

## Suggested Action

I propose stripping the overlays section completely and replacing it with a section that addresses browser extensions, native apps, and perhaps other avenues where users can choose to use the features this document outlines:

* These can learn from the user's preferences.
* They can offer better privacy controls (see [Ethical Web Principles on privacy](https://www.w3.org/TR/ethical-web-principles/#privacy)).
* They can provide more control for customizations (see [Ethical Web Principles on control](https://www.w3.org/TR/ethical-web-principles/#control)).
* They let users decide when to throw compute cycles and battery power at a challenge (see [Ethical Web Principles on sustainability](https://www.w3.org/TR/ethical-web-principles/#sustainable)).
* Unlike overlays, they can arguably support an entire device or set of devices (see [Ethical Web Principles on multi-platform](https://www.w3.org/TR/ethical-web-principles/#multi)).

Overall, replacing overlays with extensions / apps better aligns with the ethics cited in #13 and sustainability cited in #15.

It has the added benefit of keeping this note from being associated with a product offering that users and the industry widely agree is problematic.

## Background

While overlay vendors have claimed to use "AI," they have mostly only used the term "AI" in the marketing sense. Most of their remediation efforts are simple scripted features that a reasonably skilled developer could implement in JavaScript or CSS as a bookmarklet.

Looking over the use cases outlined in this document:

* Some overlays have used computer vision and LLMs to generate alternative text, but that is not unique to overlays.

* Most, if not all, of them exclude captioning from their services. Automated captioning existed before generative AI, so they could conceivably offer it today but have chosen not to.

* I am not aware of any performing automated language detection nor converting content to "plain language," let alone publicizing those as planned features.

* Instances I have reviewed of overlays adjusting color contrast appear to be brute-force methods (backplates for text, CSS overrides, etc) and don't seem to rely on computer vision nor have they generally appeared to trend that way.

* Similarly, cases I have seen of addressing headings appears to be algorithmic versus any higher-level machine learning.

* Same for visual spacing.

* I am not sure I have seen link purpose tackled, though that could be as simple as pre-fetching the destination and appending it to the link text.

* The few cases I have seen of generative sign language are (I have been told by fluent ASL speakers) not very compelling. This isn't to say it cannot improve dramatically, but I am unaware of overlay vendors expressing much interest.

All of those features could make it into overlays as "AI" in the future, of course. They could just as easily make it into browser extensions and custom native apps. However, I see neither browser extensions nor native apps as a discrete item in this document.

I believe browser extensions and native apps would be far better vehicles to replace Section 5 in the document. Partly because of the reputation of overlay products and vendors generally:

* One of the biggest vendors settled with the FTC "[for Deceptive Claims that its AI Product Could Make Websites Compliant with Accessibility Guidelines](https://www.ftc.gov/news-events/news/press-releases/2025/01/ftc-order-requires-online-marketer-pay-1-million-deceptive-claims-its-ai-product-could-make-websites)"

* That same vendor is also the [subject of a class action for its claims](https://adrianroselli.com/wp-content/uploads/2020/06/sherwin_v_accessibe_complaint.pdf) (PDF)

* One of its [peers is also the subject of a class action](https://techstartups.com/2024/12/23/userway-faces-class-action-lawsuit-over-alleged-false-accessibility-and-ada-compliance-claims/)

* They tend to be litigious when the quality of their output is challenged ([1](https://www.lalutineduweb.fr/aide-frais-avocate-proces-contre-faciliti/), [2](https://koena.net/en/koena-given-formal-notice-by-faciliti/), [3](https://adrianroselli.com/2023/05/audioeye-is-suing-me.html)), which is a significant problem when users try to improve the product.

* The foray of one vendor into [LLM code generation was so problematic](http://fuxmycode.ai/) that it became a running joke among practitioners.

* Disabled users built tools ([1](https://www.accessibyebye.org/), [2](https://sclower.github.io/accessibegone/)) to block overlays on their sites.

While the overlays section hints at this by saying overlays are "viewed as rather controversial by" disabled users, I think there is far more evidence saying they are actual barriers. The uptick in [lawsuits against sites with overlays](https://info.usablenet.com/ada-website-compliance-lawsuit-tracker) seems to support that.

I hope that's enough background detail, but I am happy to offer more.

**Comment by rscano** [comment]
Fully agree. Not only lawsuit but also [European Commission disagree about overlay usage](https://digital-strategy.ec.europa.eu/en/policies/web-accessibility).

> The legal accessibility requirements in the EU are underpinned by technical criteria specified in the harmonised European standard EN 301 549 v3.2.1. Accessibility overlays are tools that can be added to websites with the aim of improving accessibility. Overlays, or any other tools which do not ensure the website itself meets the detailed criteria of the standard, are not an appropriate solution. It is best to fix accessibility issues at their source.
> 
> Another good practice is to provide feedback on accessibility issues to those developing or maintaining digital tools, to ensure that those responsible learn about digital accessibility and avoid future problems. To ensure that websites are genuinely accessible, it is important to always involve persons with disabilities in testing.
> 
> The European Disability Forum, users with disabilities and many accessibility experts have expressed concern about the use of overlays. For more information visit:
> 
> [Joint statement by EDF & IAAP](https://www.edf-feph.org/publications/joint-statement-on-accessibility-overlays/)
> [Article by The A11Y project](https://www.a11yproject.com/posts/should-i-use-an-accessibility-overlay/)
> [Overlay Fact Sheet](https://overlayfactsheet.com/en/) signed by nearly 900 accessibility experts.

**Comment by aardrian** [comment]
> I would agree removing the section but not replacing with extensions or apps , as these are not AI doing anything.

You made fair points @UsableNet-WCAG and @nattarnoff.

Mostly I wanted to acknowledge the aspirational opportunities, but the _current_ crop of tools is lacking as you note. Even though future local installs _could_ offer users better control and privacy.

I would be ok with striking overlays completely without replacement.
